src/test/java/teammates/common/util/JsonUtilsTest.java

```diff
@@ -45,6 +45,7 @@ public void testFeedbackQuestionDetailsAdaptor_withComposedQuestionDetails_shoul
                     + "  \"feedbackSessionName\": \"testFeedbackSession\",\n"
                     + "  \"courseId\": \"testingCourse\",\n"
                     + "  \"questionDetails\": {\n"
+                    + "    \"allowRichText\": true,\n"
```

Comment:
There are multiple places where you should add the new fields to the test data :D

- Recommend to write a script to that
- I should have said this change technically requires data migration. @madanalogy we should create post-issue step for this.

Reply:
The change only affects question details for `FeedbackText` question types right?

Reply:
@xpdavid Do I alter all the tests already? It does not seem to break the existing tests so I probably would have to use IDE lookup to update all.

Reply:
@madanalogy Yes. It is part of FeedbackTextQuestionDetails. Totally contained like other attributes.

Reply:
> Do I alter all the tests already?

No. See at least `typicalDataBundle.json`. Or we can edit the test files as post-data migration step (to "simulate" the legacy data format) @madanalogy your call :D